gnc/sub8_perception/src/sub8_pcl/torpedo_board.cc

```diff
@@ -1,15 +1,28 @@
 #include <sub8_pcl/torpedo_board.hpp>
 
 Sub8TorpedoBoardDetector::Sub8TorpedoBoardDetector()
-try : frame_history("/forward_camera/image_color", 10), image_transport(nh)
+try : image_transport(nh), rviz("/visualization/torpedo_board")
 {
-  std::string img_topic = /*"/forward_camera/image_color"*/ "/stereo/left/image_raw";
-  ROS_INFO("Constructing Sub8TorpedoBoardDetector");
-  image_sub = image_transport.subscribeCamera(img_topic, 1, &Sub8TorpedoBoardDetector::image_callback, this);
-  // ROS_INFO("Subscribed to  %s ", img_topic.c_str());
+  ROS_INFO("Initializing Sub8TorpedoBoardDetector");
+  std::string img_topic_left = "/stereo/left/image_raw";
```

Comment:
Why not image_rect?

If you're using a bag that only has image_raw and camrea_info, you can run
ROS_NAMESPACE=/stereo/left rosrun image_proc image_proc
ROS_NAMESPACE=/stereo/right rosrun image_proc image_proc

To generate the rectified topics